projects/cashmere/src/lib/sass/form-field.scss

```diff
@@ -68,14 +68,14 @@ $wrapper-padding-bottom: ($error-margin-top + $line-height) * $error-font-scale;
 
     background: $white;
 
-    @include formFieldFocusTransition();
+    //@include formFieldFocusTransition();
 }
 
 @mixin hc-form-field-input-focused() {
     @include formFieldFocusedState();
-    &:not(.hc-form-field-invalid) {
-        padding: 0.5px;
-    }
+    // &:not(.hc-form-field-invalid) {
+    //     padding: 0.5px;
+    // }
```

Comment:
Please delete instead of commenting out.

Reply:
I deleted this as well.